app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,100 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>SacredArt</title>
+  <title>Sacred Art</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
+  <link href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css" rel=stylesheet>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+<nav>
+  <%= link_to image_tag("sa_logo_small.png", :alt => "Sacred Art Logo", :class => "logo"), index_path %>
```

Comment:
instead of this being built into this page, this should be a partial with a yield.